### PR TITLE
SRVCOM-1728: Update abstracts for event source docs to use Jupiter guidelines, other misc. improvements

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1728

Applies for OCP 4.6+, may need manual cherry picks for older versions due to different event source functionality in ODC.

Direct preview links:
- https://deploy-preview-44337--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-apiserversource.html
- https://deploy-preview-44337--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-pingsource.html
- https://deploy-preview-44337--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-custom-event-sources.html

### CODE REVIEW

**Comment by pierDipi** [comment]
/cc

I'll take a look tomorrow unless someone else can have a look earlier.

File: modules/serverless-containersource-guidelines.adoc [hunk 1]
```diff
@@ -6,12 +6,9 @@
 [id="serverless-containersource-guidelines_{context}"]
 = Guidelines for creating a container image
 
-* Two environment variables are injected by the container source controller: `K_SINK` and `K_CE_OVERRIDES`. These variables are resolved from the `sink` and `ceOverrides` spec, respectively.
+Two environment variables are injected by the container source controller: `K_SINK` and `K_CE_OVERRIDES`. These variables are resolved from the `sink` and `ceOverrides` spec, respectively. Event messages are sent to the sink URI specified in the `K_SINK` environment variable. The event message can be in any format; however, using the link:https://cloudevents.io/[`CloudEvent`] spec is recommended.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/serverless-containersource-guidelines.adoc]: Updated, guessing this was changed in upstream since we added it here 🙂 thanks

**Review by pierDipi — APPROVED** [review]
/lgtm

**Review by jrangelramos — COMMENTED** [review]
QE Approved

File: modules/serverless-containersource-reference.adoc [hunk 2]
```diff
@@ -75,7 +72,7 @@ spec:
 [id="serverless-containersource-reference-cloudevent-overrides_{context}"]
 == CloudEvent overrides
 
-A `ceOverrides` definition provides overrides that control the CloudEvent's output format and modifications sent to the sink
+A `ceOverrides` definition provides overrides that control the CloudEvent's output format and modifications sent to the sink. You can configure multiple different fields for the `ceOverrides` definition.
```
Comment (mburke5678) [inline modules/serverless-containersource-reference.adoc:75]: Is _multiple different_ redundant here?

File: modules/serverless-sinkbinding-reference.adoc [hunk 3]
```diff
@@ -5,8 +5,9 @@
 :_content-type: REFERENCE
 [id="serverless-sinkbinding-reference_{context}"]
 = Sink binding reference
+// this section probably needs a rewrite / restructure; feels like multiple modules maybe for a larger ref doc. Out of scope for this PR.
 
-This topic provides reference information about the configurable parameters for `SinkBinding` objects.
+You can use a `PodSpecable` object as an event source, by creating a sink binding. You can configure multiple different parameters when creating a `SinkBinding` object.
```
Comment (mburke5678) [inline modules/serverless-sinkbinding-reference.adoc:10]: Do you need the comma after _event source_?

_multiple different_ again

File: modules/serverless-sinkbinding-reference.adoc [hunk 4]
```diff
@@ -49,7 +50,7 @@ This topic provides reference information about the configurable parameters for
 [id="serverless-sinkbinding-reference-subject-parameters_{context}"]
 == Subject parameter
 
-The `Subject` parameter references the resources for which the runtime contract is augmented by binding implementations.
+The `Subject` parameter references the resources for which the runtime contract is augmented by binding implementations. You can configure multiple fields for a `Subject` definition.
```
Comment (mburke5678) [inline modules/serverless-sinkbinding-reference.adoc:53]: No _multiple different_ here. I think it reads better.

File: modules/serverless-sinkbinding-reference.adoc [hunk 5]
```diff
@@ -166,7 +166,7 @@ spec:
 [id="serverless-sinkbinding-reference-cloudevent-overrides_{context}"]
 == CloudEvent overrides
 
-A `ceOverrides` definition provides overrides that control the CloudEvent's output format and modifications sent to the sink
+A `ceOverrides` definition provides overrides that control the CloudEvent's output format and modifications sent to the sink. You can configure multiple different fields for the `ceOverrides` definition.
```
Comment (mburke5678) [inline modules/serverless-sinkbinding-reference.adoc:169]: _multiple different_ here

File: modules/apiserversource-yaml.adoc [hunk 6]
```diff
@@ -6,14 +6,14 @@
 [id="apiserversource-yaml_context"]
 = Creating an API server source by using YAML files
 
-You can use the following procedure to create an API server source by using YAML files.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe event sources declaratively and in a reproducible manner. To create an API server source by using YAML, you must create a YAML file that defines an `ApiServerSource` object, then apply it by using the `oc apply` command.
```
Comment (mburke5678) [inline modules/apiserversource-yaml.adoc:9]: Not quite sure how to fix it, but files uses seems awkward.

_Creating Knative resources by using YAML files enables you
When you create Knative resources by using YAML files, you use a declarative API ??_

Would the second sentence be a stronger lead?

File: modules/odc-creating-apiserversource.adoc [hunk 7]
```diff
@@ -6,14 +6,14 @@
 [id="odc-creating-apiserversource_{context}"]
 = Creating an API server source by using the web console
 
-You can use the following procedure to create an API server source by using the {product-title} web console.
+Using the {product-title} web console provides a streamlined and intuitive user interface to create an event source. After Knative Eventing is installed on your cluster, you can create an API server source by using the web console.
```
Comment (mburke5678) [inline modules/odc-creating-apiserversource.adoc:9]: Would the second sentence be a stronger lead?

File: modules/serverless-containersource-reference.adoc [hunk 8]
```diff
@@ -6,8 +6,7 @@
 [id="serverless-containersource-reference_{context}"]
 = Container source reference
 
-This topic provides reference information about the configurable fields for the
-`ContainerSource` object.
+You can use a container as an event source, by creating a `ContainerSource` object. You can configure multiple different parameters when creating a `ContainerSource` object.
```
Comment (mburke5678) [inline modules/serverless-containersource-reference.adoc:9]: _multiple different_

File: modules/serverless-kn-containersource.adoc [hunk 9]
```diff
@@ -6,8 +6,9 @@
 :_content-type: REFERENCE
 [id="serverless-kn-containersource_{context}"]
 = Creating and managing container sources by using the Knative CLI
+// needs to be revised as separate procedure modules; out of scope for this PR
 
-You can use the following `kn` commands to create and manage container sources:
+Using the `kn` CLI to create event sources provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn source container` commands to create  and manage container sources by using the `kn` CLI.
```
Comment (mburke5678) [inline modules/serverless-kn-containersource.adoc:11]: Would the second sentence be a stronger lead?

File: modules/serverless-odc-create-containersource.adoc [hunk 10]
```diff
@@ -6,7 +6,7 @@
 [id="serverless-odc-create-containersource_{context}"]
 = Creating a container source by using the web console
 
-You can create a container source by using the *Developer* perspective of the {product-title} web console.
+Using the {product-title} web console provides a streamlined and intuitive user interface to create an event source. After Knative Eventing is installed on your cluster, you can create a container source by using the web console.
```
Comment (mburke5678) [inline modules/serverless-odc-create-containersource.adoc:9]: Would the second sentence be a stronger lead?

File: modules/serverless-pingsource-kn.adoc [hunk 11]
```diff
@@ -7,13 +7,14 @@
 [id="serverless-pingsource-kn_{context}"]
 = Creating a ping source by using the Knative CLI
 
-You can use the following procedure to create a ping source by using the `kn` CLI.
+Using the `kn` CLI to create event sources provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn source ping create` command to create a ping source by using the `kn` CLI.
```
Comment (mburke5678) [inline modules/serverless-pingsource-kn.adoc:10]: Would the second sentence be a stronger lead?

File: modules/serverless-pingsource-odc.adoc [hunk 12]
```diff
@@ -6,7 +6,7 @@
 [id="serverless-pingsource-odc_{context}"]
 = Creating a ping source by using the web console
 
-You can use the following procedure to create a ping source by using the {product-title} web console.
+Using the {product-title} web console provides a streamlined and intuitive user interface to create an event source. After Knative Eventing is installed on your cluster, you can create a ping source by using the web console.
```
Comment (mburke5678) [inline modules/serverless-pingsource-odc.adoc:9]: Would the second sentence be a stronger lead?

File: modules/serverless-pingsource-yaml.adoc [hunk 13]
```diff
@@ -6,20 +6,41 @@
 [id="serverless-pingsource-yaml_{context}"]
 = Creating a ping source by using YAML
 
-The following sections describe how to create a basic ping source using YAML files.
+Creating Knative resources by using YAML files uses a declarative API, which enables you to describe event sources declaratively and in a reproducible manner. To create a serverless ping source by using YAML, you must create a YAML file that defines a `PingSource` object, then apply it by using `oc apply`.
```
Comment (mburke5678) [inline modules/serverless-pingsource-yaml.adoc:9]: _files uses_

Would the second sentence be a stronger lead?

File: modules/serverless-sinkbinding-kn.adoc [hunk 14]
```diff
@@ -6,13 +6,14 @@
 [id="serverless-sinkbinding-kn_{context}"]
 = Creating a sink binding by using the Knative CLI
 
-This guide describes the steps required to create a sink binding instance using `kn` commands.
+Using the `kn` CLI to create event sources provides a more streamlined and intuitive user interface over modifying YAML files directly. You can use the `kn source binding create` command to create a sink binding by using the `kn` CLI.
```
Comment (mburke5678) [inline modules/serverless-sinkbinding-kn.adoc:9]: Would the second sentence be a stronger lead?

**Comment by mburke5678** [comment]
@abrennan89 A few style preference comments that you can take as you see fit. Otherwise LGTM

File: modules/apiserversource-yaml.adoc (hunk 6 above)
Comment (abrennan89) [reply modules/apiserversource-yaml.adoc:9]: responded to similar comment in other PR, gonna leave as is for now

**Pull request merged**